v4-client-js/src/clients/modules/post.ts

```diff
@@ -159,11 +182,11 @@ export class Post {
     private async signTransaction(
       wallet: LocalWallet,
       messages: EncodeObject[],
+      account: Account,
```

Comment:
I wonder if we can make the account param optional.
and we can fallback to:
`const account = await this.account(wallet.address, undefined);`

That way we won't break existing integrations either.